e2e-test/cypress/plugins/index.js

```diff
@@ -67,8 +70,49 @@ async function awaitApiReady(baseUrl) {
   return null;
 }
 
+async function readExcelSheet({index, file}) {
+  if (fs.existsSync(file)) {
+    const workbook = new Excel.Workbook();
+    await workbook.xlsx.readFile(file);
+    return workbook.worksheets[index]["name"];
+  }
+
+  return null;
+}
+
+async function deleteFile(file) {
+  if (fs.existsSync(file)) {
+    fs.unlinkSync(file);
+    return true;
+  }
+  return false;
+}
+
+async function checkFileExists(file) {
```

Comment:
add a timeout variable that controls for how long the filesystem should be checked